corona-warn-app/cwa-app-android
# PPA Collect key submission metadata (EXPOSUREAPP-4821)

Tech spec: https://github.com/corona-warn-app/cwa-app-tech-spec/blob/40e00dfd0e4544bdf05a2ae83fa7123af86b9438/docs/spec/data-donation-client-ppa.md#key-submission-metadata


How to test:
- go through the submission flow and check whether the data donation values match your behavior

Test cases:
- with or without prior high risk warning
- with or without prior consent
- with or without withdrawal of consent
- ending the submission flow on different screens
- trigger background submission by not finishing the symptom flow
- use QR code or TAN

Missing: Unit tests
